project/Build.scala

```diff
@@ -675,6 +677,7 @@ object Build {
           // library on the compiler classpath since the non-bootstrapped one
           // may not be binary-compatible.
           "dotty-library"       -> packageBin.in(`dotty-library-bootstrapped`, Compile).value,
+          "tasty-core"          -> packageBin.in(`tasty-core-bootstrapped`, Compile).value,
```

Comment:
For the non-bootstrapped compiler this should be "tasty-core" (it's not a library on the compiler classpath so the bootstrapped version isn't needed here), tasty-core-bootstrapped should only be used in the redefinition of packageAll in bootstrapedDottyCompilerSettings.

Reply:
Thanks :)